comixed-webui/src/app/comic-books/selectors/comic-list.selectors.spec.ts

```diff
@@ -45,6 +47,7 @@ describe('Comic List Selectors', () => {
     { ...COMIC_3, lastRead: null, comicState: ComicBookState.STABLE },
     { ...COMIC_5, lastRead: null, comicState: ComicBookState.DELETED }
   ];
+  const COVER_DATE_FILTER = { month: 4, year: 2022 } as CoverDateFilter;
```

Comment:
Everywhere in the code it is year then month.

Reply:
I'll swap them about tonight and update the PR.